API.md

```diff
@@ -228,14 +228,18 @@ mapbox://styles/bobbysud/cigtw1pzy0000aam2346f7ex0
 ```javascript
 [{
   coordinates, // required. For type polyline and polygon must be an array of arrays. For type point, single array with 2 coordinates
-  type, // required. One of 'point', 'polyline' or 'polygon'
+  type, // required. One of 'point', 'view', 'polyline' or 'polygon'
   title, // optional. Title string. Appears when marker pressed
   subtitle, // optional. Subtitle string. Appears when marker pressed
   fillAlpha, // optional. number. Only for type=polygon. Controls the opacity of the polygon
   fillColor, // optional. string. Only for type=polygon. CSS color (#rrggbb). Controls the fill color of the polygon
   strokeAlpha, // optional. number. Only for type=polygon or type=polyline. Controls the opacity of the line
   strokeColor, // optional. string. Only for type=polygon or type=polyline. CSS color (#rrggbb). Controls line color.
   strokeWidth, // optional. number. Only for type=polygon or type=polyline. Controls line width.
+  alpha, // optional. number. Only for type=view. Sets the marker icon opacity. Defaults 1.0.
+  rotation, // optional. number. Only for type=view. Sets the orientation of the marker clockwise starting from the bottom.
+  visible, // optional. boolean. Only for type=view. Passing false will cause the marker to be invisible
```

Comment:
The three properties above can be implemented trivially on iOS. If you don't feel comfortable doing that, would you mind opening an issue tracking that work? In the meantime, please add a note that these four properties are currently unsupported on iOS.